apps/studio/src/utils/ajv.ts

```diff
@@ -0,0 +1,3 @@
+import Ajv from "ajv"
+
+export const ajv = new Ajv({ allErrors: true, strict: false, logger: false })
```

Comment:
maybe not related to this PR, why do we use strict false?

Reply:
cause there may be more metadata on the input than exactly the schema, which causes ajv to throw. this one need ask @dcshzj maybe is code smell.

Reply:
i think this is because we need to have extra properties on the item that ajv doesn't support. i could be wrong here thoguh